tests/kuttl/tests/dataplane-create-test/00-assert.yaml

```diff
@@ -92,11 +92,11 @@ spec:
 status:
   conditions:
   - message: Deployment not started
-    reason: NotRequested
+    reason: Requested
```

Comment:
The Deployment of the NodeSet hasn't actually been Requested in this test though, so I think in the `else` clause you added it should use `condition.NotRequestedReason`?

Reply:
We always check whether a deployment  is there or not  and it's status after a nodeset is SetupReady, so the status is requested right?

Reply:
I don't think it's Requested until an associated Deployment is created.

Reply:
Sorry, I thought I had replied to this. We `always` check if there are deployments for the NodeSet and there can be more than one, so I think the `NodeSet deployment status` is always `Requested`. If we don't agree with it we can change it later.